substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/JavaLangSubstitutions.java

```diff
@@ -708,27 +709,17 @@ private static boolean hasClassPath() {
         return true;
     }
 
-    @SuppressWarnings("unused")
-    @Substitute
-    private static URL findResource(String mn, String name) {
-        return ClassLoader.getSystemClassLoader().getResource(name);
-    }
+    @AnnotateOriginal
```

Comment:
What is the `AnnotateOriginal` for? You are not adding any annotations, so this is not going to do anything